sunrise-commander.el

```diff
@@ -908,6 +908,11 @@ Uses `save-selected-window' internally."
          (unless (kill-buffer dispose)
            (kill-local-variable 'sr-current-path-faces))))))
 
+(defun sr-assert-other ()
+  "Signal an error if we have no other pane."
+  (unless (window-live-p (sr-other))
+    (error "No other Sunrise Commander pane")))
```

Comment:
If the cause of this error is not code error but user operation, can you use `user-error` instead of `error`?